go-controller/pkg/ovn/gateway_init.go

```diff
@@ -341,6 +401,13 @@ func (oc *Controller) gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet,
 		}
 	}
 
+	if config.Gateway.DisableSNATMultipleGWs {
+		// sync stale SNAT's for the pods on this node; NOTE: egressIP SNATs are sycned in EIP controller.
```

Comment:
ack I renamed function name to `cleanupStalePodSNATs` upon @kyrtapz 's suggestion -> since technically its not a sync which suggests we do it periodically.

Also I have changed the code to always call the cleanup function and then inside the cleanup function I return immediately if disableSNATGW=false